endpoints-support.md

```diff
@@ -510,8 +546,8 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
 |<sub>/rest/users/{userName}/roles</sub>                                                  | POST     | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/users/{userName}/roles</sub>                                                  | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |     **Version**                                                                                                                         |
-|<sub>/rest/version</sub>                                                                 | GET      |                      |                      |                      |
-|     **Volume**                                                                                                                                   |
+|<sub>/rest/version</sub>                                                                 | GET      | :black_square_button: | :black_square_button: |:black_square_button:
```

Comment:
probably this will be changed, because this endpoint has merged to master branch.